repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
Sure, fine by me. Let's create a JIRA to automate the build of the dcos-hdfs (hdfs subcommand). Will you be looking into it? If so, assign it to yourself. If not, assign it to me for now.